docs/demos/podman-ai-lab-to-rhoai/podman-ai-lab-to-rhoai.md

```diff
@@ -0,0 +1,447 @@
+
+7. Upload or import the ./notebooks/Langchain-ElasticSearchVector-ingest.ipynb notebook to your workbench.
+
+    ![RHOAI Workbench Notebook](img/vector_ingest_notebook_upload.png)
+
+8. Run the first 3 cells (*Shift + Enter in each cell to run*). Make sure you can connect to your Elasticsearch cluster.
+
+    ![Notebook Execution - First Section](img/verify_elastic_connection.png)
+
+9. Continue to run through each cell while reading through what is occurring in each one. The *Create the index and ingest the documents* cell is where all of the websites and pdfs are stored with embeddings into the Elasticsearch vector database.
+
+    This cell will take a while to run. Good time for a coffee break.
+
+    ![Notebook Execution - Second Section](img/ingest_data_cell.png)
+
+10. After all of the data is stored into our vector database we can directly query it. Run the last 2 cells to make sure the data was stored successfully and we get back the results we expect.
+
+    ![Notebook Execution - Third Section](img/query_data_cell.png)
+
+## Deploy s3 Storage (Minio)
+OpenShift AI model serving has a dependency on s3 storage. We'll deploy Minio for this tutorial, but any s3 compatible storage should work. For an enterprise s3 storage solution consider [OpenShift Data Foundation](https://www.redhat.com/en/technologies/cloud-computing/openshift-data-foundation){:target="_blank"}.
+
+1. Make sure you're still logged into your cluster in your terminal. Run the below command to deploy Minio.
+    ```
+    oc apply -k ./components/minio/base
+    ```
+
+    Make sure the pod is running.
+
+    ```
+    watch oc get pods -n minio
+    ```
+
+    ![Minio Pod Status](img/minio_pod_status.png)
+
+2. Login to the Minio UI. You can find the route in either the web console or from the oc cli in your terminal. Login with *minio/minio123*. Minio contains 2 routes, an API route and UI route. Make sure you use the UI route.
+
+    ![Minio Login](img/minio_login.png)
+
+3. Create a bucket named *models* and click the **Create Bucket** button.
+
+    ![Minio bucket](img/minio_bucket.png)
+
+4. Go to **Object Browser**, select the *models* bucket you just created, and click the **Create new path** button. Name the folder path *mistral7b* and click the **Create** button.
+
+    ![Minio path](img/minio_path.png)
+
+5. Upload the Mistral7b model to the folder path you just created. You can find out where the model was downloaded if you go back to Podman AI Lab and click the **Open Model Folder** icon.
+
+    ![Model folder](img/podman_ai_model_folder.png)
+
+    In Minio click on the **Upload File** button and select the model file under the *hf.TheBloke.mistral-7b-instruct-v0.2.Q4_K_M* directory.
+
+    ![Model upload](img/minio_model_upload.png)
+
+6. If the model is uploaded successfully you should see the below screen.
+
+    ![Model upload success](img/minio_model_upload_success.png)
+
+## Create Custom Model Serving Runtime
```

Comment:
This is part of RHOAI installation, refer to the doc.

Reply:
Included the installation instructions in the product documentation and made the automated installation collapsible.